### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by ehma90**
### Portfolio Accessibility
**On this webage, We:**

- Page titles - no issues found.
- Image text alternative - resolved.
- Text heading - resolved.
- Colour contrast - no issues found.
- Resize - no issues found.
- Interaction - no issues found.
- Moving Content - no issues found.
- Multimedia - no issues found.
- The basic structure if the page - no issues found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,7 +50,7 @@ <h2>
       <section class="hero-container" id="hero-section">
         <div class="hero-section">
           <div class="hero-welcome-msg">
-            <h1 class="intro-msg">I’m</h1>
+            <h2 class="intro-msg">I’m</h2>
```
Comment (youmari) [inline index.html]: - [x] You should keep this title with `<h1>` cause it's the main title so please try change it back to h1

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @ehma90 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter error 
✔️ Great title and short summary 
✔️Github flow 

## Required Changes ♻️

- [ ] Please make sure to add a descriptive title to your portfolio some good recommendation:
    - Mark's portfolio
    - Mark full-stack webdeveloper

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ehma90) [reply index.html]: I have implemented this changes

**Review by algerina — APPROVED** [review]
Hi @uisendy , 
Good job so far!

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE
 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**